docs/_includes/jlbp-nav.html

```diff
@@ -0,0 +1,20 @@
+<nav id="jlbp-nav" class="docs-component-nav">
+  <ul class="nav-list">
+    <li class="nav-item nav-item-header">Best Practices</li>
+    {% for p in site.pages %}{% if p.jlbp -%}
+    <li class="nav-item{% if p.url == page.url %} nav-item-active{% endif %}">
+      <a href="{{ p.url | relative_url }}">
+        <span class="jlbp-number">{{ p.jlbp.id }}</span> {{ p.title }}
+      </a>
+    </li>
+    {% endif %}{% endfor -%}
+    <li class="nav-item nav-item-header">Concepts</li>
+    <li class="nav-item{% if page.url == '/concepts-index' %} nav-item-active{% endif %}">
+      <a href="/concepts-index">Concepts index</a>
```

Comment:
this is a section with one link. Either make it a single link to Concepts, or list the concept pages individually.

Reply:
I'll just list the concept pages here.